python-api/gstswitch/server.py

```diff
@@ -370,8 +481,9 @@ def gcov_flush(self):
             raise ServerProcessError('Server process does not exist')
         else:
             try:
-                print("GCOV FLUSH")
-                os.kill(self.pid, signal.SIGUSR1)
+                self.log.debug("Signaling GCOV Flush to %s", self.pid)
+                if self.pid:
+                    os.kill(self.pid, signal.SIGUSR1)
```

Comment:
We should be using `subprocess.Popen.send_signal` (IE `self.process.send_signal(signal.SIGUSR1)`)